docs/install.rst

```diff
@@ -149,17 +161,13 @@ issues, please submit a detailed issue notice on `GitHub
 <https://github.com/freedomofpress/securedrop/issues/new>`__ or send
 an email to securedrop@freedom.press.
 
-.. note:: The SecureDrop install process configures a custom Linux
```

Comment:
was this moved somewhere that i missed? we do need to advertise this for GPL compliance reasons

Reply:
Does it need to be in the install page? It isn't going relevant to the install process for most people. It is still available in `development/apt_repo.rst` and could probably be added elsewhere too.

Reply:
I'm just gonna add this back in in the interests of time, will revisit at a later date.